### PR TITLE
create 404 page

### LINKED ISSUE #64: 404 Page
image

### PR DESCRIPTION
**Pull request by shirleyduong**
image

### CODE REVIEW

File: src/app/not-found.tsx [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+import { links } from "@/data/Footer";
+
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="flex h-[820px] w-full flex-col items-center justify-center bg-gradient-to-b from-hlg-dark-gray via-hlg-dark-gray to-hlg-red-200">
+      <Image
+        src={Logo}
+        alt="Highlander Gloves Logo"
+        className="absolute left-8 size-[620px] translate-y-20 transform items-end"
```
Comment (SafeDuck) [inline src/app/not-found.tsx:13]: is it possible to use % values for `size`?

File: src/app/not-found.tsx [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+      />
+      <div className="flex flex-col items-center justify-center pt-20">
+        <div className="relative flex items-center justify-center rounded-3xl bg-hlg-red-200 px-28 py-8 text-3xl">
+          <p className="absolute translate-x-2 translate-y-2 self-center font-archivo-black text-9xl text-black">
+            404
+          </p>
+          <p className="relative font-archivo-black text-9xl">404</p>
+        </div>
+        <p className="relative pb-10 pt-4 font-archivo-narrow text-[130px]">
```
Comment (SafeDuck) [inline src/app/not-found.tsx]: use `rem` for `text` size

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/app/not-found.tsx (hunk 1 above)
Comment (shirleyduong) [reply src/app/not-found.tsx:13]: when I try using %, it makes the photo compressed

File: src/app/not-found.tsx (hunk 2 above)
Comment (shirleyduong) [reply src/app/not-found.tsx]: do you mean like this? `text-[8.125rem]`

File: src/app/not-found.tsx (hunk 1 above)
Comment (SafeDuck) [reply src/app/not-found.tsx:13]: ok np

File: src/app/not-found.tsx (hunk 2 above)
Comment (SafeDuck) [reply src/app/not-found.tsx]: yes or use `em` up to you

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**